### ISSUE TITLE
Question about backbone's training strategy

### ISSUE THREAD

**Issue by onotoa** [wyczzy/StealthDiffusion#1]
Hi~Thanks for your great works!
In your codes, the backbones are trained on GenImage, applying CNNSpot's earlystop training strategy.
But according to your paper on openreview.net, they are trained for 10 epochs. I wonder which one is correct. 
![image](https://github.com/user-attachments/assets/ec811c9b-27b2-4efb-bd00-8894f66364c7)

Thanks!

**Comment by wyczzy** [comment]
Thank you for your attention to our work. We follow the approach of CNNSpot and have applied CNNSpot's early stopping training strategy. We have set the total number of training epochs to 10. As configured in our code, the default patience for the early stop strategy is set to 5, meaning that if the accuracy on the validation set does not improve for five consecutive epochs, we will proceed with our subsequent strategies, such as adjusting the learning rate or stopping the training altogether, and selecting the best weights based on the validation set. You can also use our code to modify these parameters and conduct training according to your needs.

**Comment by onotoa** [comment]
Thanks for your rapid reply!
Do you use the whole validation set(include your 800-image dataset for attack) during training? Or you split 800-image dataset for evaluate your attack before your training?

**Comment by wyczzy** [comment]
To ensure the performance of the trained classifier, we use the whole validation set (including your 800-image dataset for attack) during training. All our classifiers achieved over 96% accuracy and 98% average precision on the GenImage validation set.
